### PR TITLE
 - did some more refactoring

### PR DESCRIPTION
**Pull request by rakeshongithub**
- [x] Updated ssg and ssr scripts to generate project
 - [x] removing node_module and lock file default generated by turbo repo and installing a fresh one
 - [x] update some log messages
 - [x]  Updated SSG and SSR package.json files
 - [x]  created common package.json under common folder to separate out dependencies between ssg/ssr vs microApp

### CODE REVIEW

**Comment by rakeshongithub** [comment]
@vinodloha  Update the package.json and implemented the review comments.

File: templates/common/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "scripts": {
+      "test:watch": "{commandType} test -- --watch",
+      "test": "jest -c ./config/jest/jest.config.js",
+      "generate": "plop --plopfile generators/index.js && {commandType} pretty-quick",
```
Comment (vinodloha) [inline templates/common/package.json:5]: Lets start using our component generator here and delete the generator specific things.

File: templates/common/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "bundlesize": "bundlesize --config ./config/bundlesize.json",
+      "analyze": "cross-env ANALYZE=true && {commandType} build",
+      "perf": "docker-compose -f ./config/performance/sitespeed-compose.yml run sitespeed.io  --config sitespeed.config.json --budget.configPath sitespeed.budget.config.json --graphite.host=graphite urls.txt",
+      "pretty-quick": "pretty-quick"
```
Comment (vinodloha) [inline templates/common/package.json:9]: We need to remove pretty-quick as turbo-repo brings prettier along.

File: templates/common/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "dependencies": {
+        "markdown-to-jsx": "^7.1.3",
+        "react": "*",
+        "react-dom": "*",
+        "styled-jsx": "4.0.1",
```
Comment (vinodloha) [inline templates/common/package.json:15]: Remove  "styled-jsx": "4.0.1",

**Review by vinodloha — APPROVED** [review]

**Pull request merged**